server/store/datastore/migration/024_unsanitize_org_and_user_names.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2025 Woodpecker Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package migration
+
+import (
+	"fmt"
+
+	"src.techknowlogick.com/xormigrate"
+	"xorm.io/xorm"
+)
+
+var unSanitizeOrgAndUserNames = xormigrate.Migration{
+	ID: "unsanitize-org-and-user-names",
+	MigrateSession: func(sess *xorm.Session) (err error) {
+		type user struct {
+			Login string `xorm:"TEXT 'login'"`
+		}
+
+		type org struct {
+			Name string `xorm:"TEXT 'name'"`
+		}
+
+		if err := sess.Sync(new(user), new(org)); err != nil {
+			return fmt.Errorf("sync new models failed: %w", err)
+		}
+
+		// get all users
+		var users []*user
```

Comment:
Should use pagination to allow smooth migrations on larger instances

Reply:
Even large instances should have only xxxx users (e.g. Codeberg). Getting this amount of users without pagination in one call should be acceptable? I would worry more about pagination for getting all pipeline logs or similar but I think we don't need pagination here. 

I only know the shared_utils.Paginate() but I don't wanna make the code more complex than it really needs to be.

Reply:
That said, feel free to add pagination if you think it is really needed and/or you know directly what the best way would be.